src/main/java/spoon/support/compiler/jdt/PositionBuilder.java

```diff
@@ -539,9 +538,6 @@ private void setModifiersPosition(CtModifiable e, int start, int end) {
 			}
 			start = o2;
 		}
-		if (explicitModifiersByName.size() > 0) {
-			throw new SpoonException("Position of CtExtendedModifiers: [" + String.join(", ", explicitModifiersByName.keySet()) + "] not found in " + String.valueOf(contents, start, end - start));
-		}
```

Comment:
Is there a legitimate reason why `final` modifier would remain in `explicitModifiersByName` here?